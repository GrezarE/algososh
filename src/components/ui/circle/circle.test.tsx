import React from "react";
import { render, screen, fireEvent } from "@testing-library/react";

import { Circle } from "./circle";
import { ElementStates } from "../../../types/element-states";

it("Круг без буквы", () => {
  const circle = render(<Circle />);
  expect(circle).toMatchSnapshot();
});

it("Круг с буквами", () => {
  const circle = render(<Circle letter="Круг" />);
  expect(circle).toMatchSnapshot();
});

it("Круг с head", () => {
  const circle = render(<Circle head="head" />);
  expect(circle).toMatchSnapshot();
});

it("Круг react-элементом в head", () => {
  const circle = render(<Circle head={React.createElement("p")} />);
  expect(circle).toMatchSnapshot();
});

it("Круг c tail", () => {
  const circle = render(<Circle tail="tail" />);
  expect(circle).toMatchSnapshot();
});

it("Круг react-элементом в tail", () => {
  const circle = render(<Circle tail={React.createElement("div")} />);
  expect(circle).toMatchSnapshot();
});

it("Круг с index", () => {
  const circle = render(<Circle index={1} />);
  expect(circle).toMatchSnapshot();
});

it("Маленький круг", () => {
  const circle = render(<Circle isSmall={true} />);
  expect(circle).toMatchSnapshot();
});

it("Круг в состоянии базовом", () => {
  const circle = render(<Circle state={ElementStates.Default} />);
  expect(circle).toMatchSnapshot();
});

it("Круг в состоянии изменяемом", () => {
  const circle = render(<Circle state={ElementStates.Changing} />);
  expect(circle).toMatchSnapshot();
});

it("Круг в состоянии изменённом", () => {
  const circle = render(<Circle state={ElementStates.Modified} />);
  expect(circle).toMatchSnapshot();
});
