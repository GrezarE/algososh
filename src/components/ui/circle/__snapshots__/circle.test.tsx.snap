// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Круг c tail 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="content "
      >
        <div
          class="text text_type_input text_color_input mb-4 absolute head element"
        />
        <div
          class="circle   default"
        >
          <p
            class="text text_type_circle text_color_input letter"
          />
        </div>
        <p
          class="text text_type_input text_color_input mt-4 absolute index"
        />
        <div
          class="text text_type_input text_color_input mt-4 absolute tail30 string"
        >
          tail
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="content "
    >
      <div
        class="text text_type_input text_color_input mb-4 absolute head element"
      />
      <div
        class="circle   default"
      >
        <p
          class="text text_type_circle text_color_input letter"
        />
      </div>
      <p
        class="text text_type_input text_color_input mt-4 absolute index"
      />
      <div
        class="text text_type_input text_color_input mt-4 absolute tail30 string"
      >
        tail
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Круг react-элементом в head 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="content "
      >
        <div
          class="text text_type_input text_color_input mb-4 absolute head element"
        >
          <p />
        </div>
        <div
          class="circle   default"
        >
          <p
            class="text text_type_circle text_color_input letter"
          />
        </div>
        <p
          class="text text_type_input text_color_input mt-4 absolute index"
        />
        <div
          class="text text_type_input text_color_input mt-4 absolute tail30 element"
        />
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="content "
    >
      <div
        class="text text_type_input text_color_input mb-4 absolute head element"
      >
        <p />
      </div>
      <div
        class="circle   default"
      >
        <p
          class="text text_type_circle text_color_input letter"
        />
      </div>
      <p
        class="text text_type_input text_color_input mt-4 absolute index"
      />
      <div
        class="text text_type_input text_color_input mt-4 absolute tail30 element"
      />
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Круг react-элементом в tail 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="content "
      >
        <div
          class="text text_type_input text_color_input mb-4 absolute head element"
        />
        <div
          class="circle   default"
        >
          <p
            class="text text_type_circle text_color_input letter"
          />
        </div>
        <p
          class="text text_type_input text_color_input mt-4 absolute index"
        />
        <div
          class="text text_type_input text_color_input mt-4 absolute tail30 element"
        >
          <div />
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="content "
    >
      <div
        class="text text_type_input text_color_input mb-4 absolute head element"
      />
      <div
        class="circle   default"
      >
        <p
          class="text text_type_circle text_color_input letter"
        />
      </div>
      <p
        class="text text_type_input text_color_input mt-4 absolute index"
      />
      <div
        class="text text_type_input text_color_input mt-4 absolute tail30 element"
      >
        <div />
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Круг без буквы 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="content "
      >
        <div
          class="text text_type_input text_color_input mb-4 absolute head element"
        />
        <div
          class="circle   default"
        >
          <p
            class="text text_type_circle text_color_input letter"
          />
        </div>
        <p
          class="text text_type_input text_color_input mt-4 absolute index"
        />
        <div
          class="text text_type_input text_color_input mt-4 absolute tail30 element"
        />
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="content "
    >
      <div
        class="text text_type_input text_color_input mb-4 absolute head element"
      />
      <div
        class="circle   default"
      >
        <p
          class="text text_type_circle text_color_input letter"
        />
      </div>
      <p
        class="text text_type_input text_color_input mt-4 absolute index"
      />
      <div
        class="text text_type_input text_color_input mt-4 absolute tail30 element"
      />
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Круг в состоянии базовом 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="content "
      >
        <div
          class="text text_type_input text_color_input mb-4 absolute head element"
        />
        <div
          class="circle   default"
        >
          <p
            class="text text_type_circle text_color_input letter"
          />
        </div>
        <p
          class="text text_type_input text_color_input mt-4 absolute index"
        />
        <div
          class="text text_type_input text_color_input mt-4 absolute tail30 element"
        />
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="content "
    >
      <div
        class="text text_type_input text_color_input mb-4 absolute head element"
      />
      <div
        class="circle   default"
      >
        <p
          class="text text_type_circle text_color_input letter"
        />
      </div>
      <p
        class="text text_type_input text_color_input mt-4 absolute index"
      />
      <div
        class="text text_type_input text_color_input mt-4 absolute tail30 element"
      />
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Круг в состоянии изменяемом 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="content "
      >
        <div
          class="text text_type_input text_color_input mb-4 absolute head element"
        />
        <div
          class="circle   changing"
        >
          <p
            class="text text_type_circle text_color_input letter"
          />
        </div>
        <p
          class="text text_type_input text_color_input mt-4 absolute index"
        />
        <div
          class="text text_type_input text_color_input mt-4 absolute tail30 element"
        />
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="content "
    >
      <div
        class="text text_type_input text_color_input mb-4 absolute head element"
      />
      <div
        class="circle   changing"
      >
        <p
          class="text text_type_circle text_color_input letter"
        />
      </div>
      <p
        class="text text_type_input text_color_input mt-4 absolute index"
      />
      <div
        class="text text_type_input text_color_input mt-4 absolute tail30 element"
      />
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Круг в состоянии изменённом 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="content "
      >
        <div
          class="text text_type_input text_color_input mb-4 absolute head element"
        />
        <div
          class="circle   modified"
        >
          <p
            class="text text_type_circle text_color_input letter"
          />
        </div>
        <p
          class="text text_type_input text_color_input mt-4 absolute index"
        />
        <div
          class="text text_type_input text_color_input mt-4 absolute tail30 element"
        />
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="content "
    >
      <div
        class="text text_type_input text_color_input mb-4 absolute head element"
      />
      <div
        class="circle   modified"
      >
        <p
          class="text text_type_circle text_color_input letter"
        />
      </div>
      <p
        class="text text_type_input text_color_input mt-4 absolute index"
      />
      <div
        class="text text_type_input text_color_input mt-4 absolute tail30 element"
      />
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Круг с head 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="content "
      >
        <div
          class="text text_type_input text_color_input mb-4 absolute head string"
        >
          head
        </div>
        <div
          class="circle   default"
        >
          <p
            class="text text_type_circle text_color_input letter"
          />
        </div>
        <p
          class="text text_type_input text_color_input mt-4 absolute index"
        />
        <div
          class="text text_type_input text_color_input mt-4 absolute tail30 element"
        />
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="content "
    >
      <div
        class="text text_type_input text_color_input mb-4 absolute head string"
      >
        head
      </div>
      <div
        class="circle   default"
      >
        <p
          class="text text_type_circle text_color_input letter"
        />
      </div>
      <p
        class="text text_type_input text_color_input mt-4 absolute index"
      />
      <div
        class="text text_type_input text_color_input mt-4 absolute tail30 element"
      />
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Круг с index 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="content "
      >
        <div
          class="text text_type_input text_color_input mb-4 absolute head element"
        />
        <div
          class="circle   default"
        >
          <p
            class="text text_type_circle text_color_input letter"
          />
        </div>
        <p
          class="text text_type_input text_color_input mt-4 absolute index"
        >
          1
        </p>
        <div
          class="text text_type_input text_color_input mt-4 absolute tail60 element"
        />
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="content "
    >
      <div
        class="text text_type_input text_color_input mb-4 absolute head element"
      />
      <div
        class="circle   default"
      >
        <p
          class="text text_type_circle text_color_input letter"
        />
      </div>
      <p
        class="text text_type_input text_color_input mt-4 absolute index"
      >
        1
      </p>
      <div
        class="text text_type_input text_color_input mt-4 absolute tail60 element"
      />
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Круг с буквами 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="content "
      >
        <div
          class="text text_type_input text_color_input mb-4 absolute head element"
        />
        <div
          class="circle   default"
        >
          <p
            class="text text_type_circle text_color_input letter"
          >
            Круг
          </p>
        </div>
        <p
          class="text text_type_input text_color_input mt-4 absolute index"
        />
        <div
          class="text text_type_input text_color_input mt-4 absolute tail30 element"
        />
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="content "
    >
      <div
        class="text text_type_input text_color_input mb-4 absolute head element"
      />
      <div
        class="circle   default"
      >
        <p
          class="text text_type_circle text_color_input letter"
        >
          Круг
        </p>
      </div>
      <p
        class="text text_type_input text_color_input mt-4 absolute index"
      />
      <div
        class="text text_type_input text_color_input mt-4 absolute tail30 element"
      />
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Маленький круг 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="content "
      >
        <div
          class="text text_type_input text_color_input mb-4 absolute head element"
        />
        <div
          class="circle  small default"
        >
          <p
            class="text text_type_circle text_color_input letter"
          />
        </div>
        <p
          class="text text_type_input text_color_input mt-4 absolute index"
        />
        <div
          class="text text_type_input text_color_input mt-4 absolute tail30 element"
        />
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="content "
    >
      <div
        class="text text_type_input text_color_input mb-4 absolute head element"
      />
      <div
        class="circle  small default"
      >
        <p
          class="text text_type_circle text_color_input letter"
        />
      </div>
      <p
        class="text text_type_input text_color_input mt-4 absolute index"
      />
      <div
        class="text text_type_input text_color_input mt-4 absolute tail30 element"
      />
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
